src/main/java/org/folio/search/service/setter/linkeddata/authority/LinkedDataAuthorityLccnProcessor.java

```diff
@@ -1,37 +1,24 @@
 package org.folio.search.service.setter.linkeddata.authority;
 
-import static java.util.stream.Collectors.toCollection;
-import static org.folio.search.domain.dto.LinkedDataAuthorityIdentifiersInner.TypeEnum.LCCN;
-
-import java.util.Collections;
-import java.util.LinkedHashSet;
-import java.util.Objects;
-import java.util.Optional;
 import java.util.Set;
-import lombok.RequiredArgsConstructor;
 import org.folio.search.domain.dto.LinkedDataAuthority;
-import org.folio.search.domain.dto.LinkedDataAuthorityIdentifiersInner;
 import org.folio.search.service.lccn.LccnNormalizer;
 import org.folio.search.service.setter.FieldProcessor;
+import org.folio.search.service.setter.linkeddata.common.LinkedDataLccnProcessor;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
 
 @Component
-@RequiredArgsConstructor
 public class LinkedDataAuthorityLccnProcessor implements FieldProcessor<LinkedDataAuthority, Set<String>> {
 
-  private final LccnNormalizer lccnNormalizer;
+  private final LinkedDataLccnProcessor linkedDataLccnProcessor;
+
+  public LinkedDataAuthorityLccnProcessor(@Autowired LccnNormalizer lccnNormalizer) {
```

Comment:
@Autowired is not necessary in this case, is it?

Reply:
right, removed it, thank you!